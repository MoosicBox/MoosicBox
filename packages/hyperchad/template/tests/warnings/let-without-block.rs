use hyperchad_template::container;

fn main() {
    container! {
        span.@let x = 1; {
            (x)
        }
    };
}
