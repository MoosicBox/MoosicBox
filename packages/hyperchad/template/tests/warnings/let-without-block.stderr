error: `@let` bindings are only allowed inside blocks
 --> tests/warnings/let-without-block.rs:5:14
  |
5 |         span.@let x = 1; {
  |              ^^^^^^^^^^^

error[E0425]: cannot find value `x` in this scope
 --> tests/warnings/let-without-block.rs:6:14
  |
6 |               (x)
  |                ^ help: a function with a similar name exists: `fx`
  |
 ::: src/lib.rs
  |
  | / pub fn fx<T>(content: T) -> actions::ActionEffect
  | | where
  | |     T: IntoActionEffect,
  | |________________________- similarly named function `fx` defined here
